plugins/inputs/vsphere/endpoint.go

```diff
@@ -1159,7 +1159,7 @@ func (e *Endpoint) collectChunk(ctx context.Context, pqs queryChunk, res *resour
 				}
 				count++
 
-				// Update highwater marks
+				// Update hiwater marks
```

Comment:
This is the normal `hiwater` item. I'm also fine with writing out `high water` or dropping this change. `highwater` is just inconsistent with the naming style of the variables.